.travis.yml

```diff
@@ -20,8 +20,10 @@ install:
   - pip install --upgrade quantities future
   - pip install pytest pytest-cov coveralls
   - pip install h5py_wrapper
+  - pip install mypy
 script:
   - pytest --cov=dicthash
+  - mypy dicthash/dicthash.py
```

Comment:
maybe change order and make conditional? if type checks fail there's no need to execute tests

Reply:
That is possible but I still think that it would be good to check both. These checks are (mostly) orthogonal so why not get the full information? The only reason to do this would be to save computational resources but our code is pretty cheap to run, so I think always running both should be fine.